account_ecotax/views/account_move_view.xml

```diff
@@ -0,0 +1,167 @@
+-->
+<odoo>
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[last()]"
+                position="after"
+            >
+                <field
+                    name="subtotal_ecotax"
+                    optional="show"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                />
+                <field
+                    name="ecotax_amount_unit"
+                    optional="hide"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                />
+                <button
+                    name="edit_ecotax_lines"
+                    type="object"
+                    string="Ecotax"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                />
+                <field name="ecotax_line_ids" invisible="1">
+                    <tree editable="bottom">
+                        <field name="classification_id" />
+                        <field name="force_amount_unit" />
+                        <field name="amount_unit" />
+                        <field name="amount_total" />
+                    </tree>
+                </field>
+            </xpath>
+            <xpath
+                expr="//field[@name='invoice_line_ids']/form/sheet/field[@name='name']"
+                position="after"
+            >
+                <field
+                    name="ecotax_line_ids"
+                    colspan="2"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                >
+                    <tree editable="bottom">
+                        <field name="classification_id" />
+                        <field name="force_amount_unit" />
+                        <field name="amount_unit" />
+                        <field name="amount_total" />
+                    </tree>
+                </field>
+                <field
+                    name="subtotal_ecotax"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                />
+                <field
+                    name="ecotax_amount_unit"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt')"
+                />
+            </xpath>
+            <xpath expr="//field[@name='tax_totals']" position="after">
+                    <field
+                    name="amount_ecotax"
+                    class="oe_subtotal_footer_separator"
+                    invisible="parent.move_type not in ('out_invoice', 'out_refund', 'out_receipt') or amount_ecotax == 0"
```

Comment:
```suggestion
                    invisible="move_type not in ('out_invoice', 'out_refund', 'out_receipt') or amount_ecotax == 0"
```
This field is not defined in a list view, so `parent` is undefined and makes it impossible to open the `account.move` form view.

Reply:
See: https://github.com/akretion/account-fiscal-rule/pull/10

Reply:
I have merged it some times ago